project1/css/index.css

```diff
@@ -52,13 +49,124 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html, body {    
+    display: flex;
     font-family: 'Titillium Web', sans-serif;
+    align-content: center;
+    justify-content: center;
+    flex-wrap: wrap;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
-    margin-bottom: 10px;
 }
+
+.container {
+  width: 800px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  height: 90px;
+}
+
+header nav {
+  width: 500px;
+  flex-grow: 1;
+}
+
+header nav ul {
+  display: flex;
+  justify-content: space-around;
+}
+
+header nav ul li a {
+  color: gray;
+  text-decoration: none;
+}
+
+header nav ul li a:hover {
+  text-decoration: underline;
+}
+
+header .logo {
+	height: 38px;
+	vertical-align: top;
+}
+
+.jumbotron {
+  display: flex;
+  justify-content: space-around;
```

Comment:
👉2 spaces here, last code block was tabbed and next code block is 4 spaces. Need consistency!